firefox-ios/Client/Frontend/Home/JumpBackIn/Data/JumpBackInDataAdaptor.swift

```diff
@@ -112,6 +113,7 @@ actor JumpBackInDataAdaptorImplementation: JumpBackInDataAdaptor, FeatureFlaggab
                 await self.delegate?.didLoadNewData()
             }
         }
+        delegate?.reloadView()
```

Comment:
You can achieve the same effect without adding a new delegate method by calling self.delegate?.didLoadNewData() here instead and removing it from the individual group tasks.

Reply:
thanks for the note and checking out my draft! 😃 Yeah I was originally wondering why it was called within each task, so thanks for confirming! Going to spend some time better understanding the tests and jumpback in logic. While you are here, I also don't see us using the `recentGroups` and we return `nil` for `updateGroupsData()`. Any issues removing it?

Reply:
Looks like it's an artifact from a previous implementation of tab grouping that we had but has since been removed. It must have been missed in the clean up, should be safe to kill it.

Reply:
thank you, removed!